src/plat/fvp/config.cmake

```diff
@@ -12,7 +12,6 @@ if(KernelPlatformFVP)
     declare_seL4_arch(aarch64)
     set(KernelArmCortexA57 ON)
     set(KernelArchArmV8a ON)
-    set(KernelArmGicV3 ON)
```

Comment:
I think (but could be wrong) that some, but not all ARMv8A have a GIC V3?

That's why I made it an explicit config option, rather than something more generic. (Plus also I think the .bf files confused me, so I didn't really know what I was doing).

If you're happy making GICv3 support same as 64-bit + Hyp support that works for me.

Reply:
There's an argument that specifies the interrupt controller version already in `declare_default_headers(INTERRUPT_CONTROLLER arch/machine/gic_v3.h)` which adds the header.  This header then includes the generated headers from .bf.

Reply:
OK, makes sense.